docs/administration-guide/gaffer-deployment/gremlin.md

```diff
@@ -55,41 +51,60 @@ flowchart LR
     --> D(Existing Gaffer Instance)
 ```
 
-To establish this connection three configuration files are required:
+To establish this connection you can make use of the existing `gaffer-gremlin`
+OCI image which is an extension of the existing `gremlin-server` image. This
+provides the Tinkerpop library to enable connecting to Gaffer graphs as well as
```

Comment:
Shouldn't this be `Tinkerpop library to allow users to` or `Tinkerpop library which allows users to` instead of `Tinkerpop library to allows users to`?